README.md

```diff
@@ -171,7 +171,7 @@ If you have a feature to add or a bug to fix please feel free to open an issue o
 - Add support for more notifications
 - Package this up for Debian/Arch/Flatpak etc.
 
-## Windows support ETA wen?
+## Windows support ETA, when?
```

Comment:
Haha, this is an insider from the exploit/jailbreak community :p 
https://www.urbandictionary.com/define.php?term=wen%20eta

But we can merge it anyway.